### ISSUE TITLE
Slight error in lift_env reward manager

### ISSUE THREAD

**Issue by adamwhats** [NVIDIA-Omniverse/Orbit#34]
https://github.com/NVIDIA-Omniverse/Orbit/blob/d75e7ff63bf08b558266d2eff42fd4db15d29d6b/source/extensions/omni.isaac.orbit_envs/omni/isaac/orbit_envs/manipulation/lift/lift_env.py#L399-L401

Looks like this missing the exp-kernel, assuming it should be something along the lines of:
```
  def reaching_object_position_exp(self, env: LiftEnv, sigma: float):
      """Penalize end-effector tracking position error using exp-kernel."""
      error = torch.sum(torch.square(env.robot.data.ee_state_w[:, 0:3] - env.object.data.root_pos_w), dim=1)
      return torch.exp(-error / sigma)
```

**Comment by Mayankm96** [comment]
Hi, yes this was an issue. 

I made a PR yesterday #33 that tunes the rewards for the lift environment to work with rsl_rl library. Haven't merged it yet. Need some time to tune it for rl_games and sb3 too.

**Comment by adamwhats** [comment]
Great news, look forward to the merge!
